docker/src/main/java/brooklyn/entity/container/docker/DockerHostSshDriver.java

```diff
@@ -289,13 +289,25 @@ public String getVersion() {
     }
 
     private String installDockerOnUbuntu() {
-        String version = getVersion();
-        log.debug("Installing Docker version {} on Ubuntu", version);
+        final String dockerVersion = getVersion();
+        final String ubuntuVersion = getMachine().getMachineDetails().getOsDetails().getVersion();
+
```

Comment:
According to Oracle's programming style, this is correct:
"One blank line should always be used in the following circumstances:
Between the local variables in a method and its first statement
..
Between logical sections inside a method to improve readability"
http://www.oracle.com/technetwork/java/javase/documentation/codeconventions-141388.html